src/bsim/winter2021/Bacterium.java

```diff
@@ -51,6 +53,8 @@ public Bacterium(BSim sim, Vector3d px1, Vector3d px2){
         this.origin_id = id;
         this.parent_id = -1;
         lifetime = 0;
+        toxintime=0;
+        contact=0;        

```

Comment:
Remove these from Bacterium.java; these should only be associated with certain types of bacteria